front/public/locales/fr/errors.json

```
@@ -1,11 +1,175 @@
+    },
+    "redis": {
+      "Url": "Url invalide '{{url}}'"
+    },
+    "rollingstocks": {
+      "BasePowerClassEmpty": "La classe de puissance par défaut ne peut être vide",
+      "CannotCreateCompoundImage": "Impossible de créer l'image composée",
+      "CannotReadImage": "Impossible de lire l'image",
+      "NameAlreadyUsed": "Un matériel roulant avec le nom '{{name}}' existe déjà",
+      "NotFound": "Matériel roulant '{{rolling_stock_id}}' non trouvé",
+      "RollingStockIsLocked": "Matériel roulant '{{rolling_stock_id}}' est verrouillé",
+      "RollingStockIsUsed": "Matériel roulant '{{rolling_stock_id}}' est occupé"
+    },
+    "scenario": {
+      "NotFound": "Scénario non trouvé"
+    },
+    "search": {
+      "ArgMissing": "Argument de type {{expected}} manquant à la position {{arg_pos}}",
+      "ArgTypeMismatch": "Argument de type {{expected}} attendu à la position {{arg_pos}}, mais {{actual}} reçu",
+      "EmptyArray": "Les tableaux vides sont interdits",
+      "IntegerConversion": "Impossible de convertir en 'i64'",
+      "InvalidColumnName": "Nom de colonne invalide",
+      "InvalidFunctionIdentifier": "L'identifiant de la fonction doit être une chaîne de caractères",
+      "InvalidSyntax": "Syntaxe invalide",
+      "ObjectType": "Le type de l'objet est invalide",
+      "QueryAst": "Une requête de type booléen est attendue",
+      "RuntimeTypeCheckFail": "Type attendu {{expected}}, mais reçu '{{value}}' de type {{actual}} à la place",
+      "UndefinedFunction": "Fonction non définie",
+      "UndefinedOverload": "Aucune surcharge appropriée",
+      "UnexpectedArg": "Argument inattendu trouvé",
+      "UnexpectedColummn": "Colonne inattendue",
+      "UnexpectedErsatz": "Une valeur de type {{expected}} était attendue, mais '{{value}}' trouvé",
+      "VariadicArgTypeMismatch": "Un argument variadique de type {{expected}} était attendu, mais {{actual}} trouvé"
+    },
+    "single_simulation": {
+      "ElectricalProfileSetNotFound": "Profil électrique '{{electrical_profile_set_id}}' non trouvé",
+      "PathNotFound": "Chemin '{{path_id}}' non trouvé",
+      "RollingStockNotFound": "Matériel roulant '{{rolling_stock_id}}' non trouvé",
+      "WrongCoreResponseFormat": "Mauvais format de réponse reçu de Core"
+    },
+    "sprites": {
+      "FileNotFound": "Fichier non trouvé",
+      "UnknownSignalingSystem": "Système de signalisation inconnu"
+    },
+    "stdcm": {
+      "InfraNotFound": "Infrastructure {{infra_id}} non trouvée"
+    },
+    "study": {
+      "NotFound": "Etude '{{study_id}}' non trouvée",
+      "StartDateAfterEndDate": "La date de début de l'étude doit commencer avant sa date de fin"
+    },
+    "timetable": {
+      "InfraNotLoaded": "L'infrastructure '{{infra_id}}' n'est pas chargée",
+      "NotFound": "Grille horaire '{{timetable_id}}' non trouvée"
+    },
+    "train_schedule": {
+      "BatchShouldHaveSameTimetable": "Le lot doit avoir une circulation identique",
```

My bad for this one, I mixed up the train schedule and timetable
```suggestion
      "BatchShouldHaveSameTimetable": "Le lot doit avoir une grille horaire identique",
```